website_event_snippet_calendar/templates/snippets.xml

```diff
@@ -0,0 +1,83 @@
+        <div
+            t-ignore="true"
+            class="s_event_calendar o_not_editable css_editable_mode_hidden"
+        >
+            <t t-call="website_event_snippet_calendar.loading" />
+        </div>
+    </template>
+    <template id="s_event_list" name="Events List">
+        <div class="o_not_editable css_non_editable_mode_hidden alert alert-info">
+            <p>Event list visible only when you save.</p>
+            <p>It will display up to the closest <b class="js_amount">4</b> events.</p>
+        </div>
+        <div
+            t-ignore="true"
+            data-date-format="LLL"
+            class="s_event_list o_not_editable css_editable_mode_hidden"
+        >
+            <t t-call="website_event_snippet_calendar.loading" />
+        </div>
+    </template>
+    <template id="s_event_calendar_list" name="Events Calendar and List">
+        <section class="oe_snippet_body s_event_calendar_list">
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-12 mt32 text-center">
+                        <h3>Events</h3>
+                    </div>
+                    <div class="col-md-4 mt16 mb16">
+                        <h4>Events Calendar</h4>
+                        <t t-call="website_event_snippet_calendar.s_event_calendar" />
+                    </div>
+                    <div class="col-md-8 mt16 mb16">
+                        <h4>List</h4>
+                        <t t-call="website_event_snippet_calendar.s_event_list" />
+                    </div>
+                    <div class="col-md-12 mb32 text-right">
+                        <a href="/event" class="btn btn-info">All events</a>
+                    </div>
+                </div>
+            </div>
+        </section>
+    </template>
+    <!-- Add snippets to menu -->
+    <template id="snippets" inherit_id="website.snippets">
+        <xpath expr="//*[@id='snippet_feature']//*[hasclass('o_panel_body')]">
+            <t
+                t-snippet="website_event_snippet_calendar.s_event_calendar_list"
+                t-thumbnail="/website_event_snippet_calendar/static/description/icon.png"
+            />
+        </xpath>
+    </template>
+    <!-- Add snippets options -->
+    <template id="snippet_options" inherit_id="website.snippet_options">
+        <xpath expr=".">
+            <!-- Allow to add or remove events from the list -->
+            <div
+                data-js="website_event_snippet_calendar_list"
+                data-selector=".s_event_calendar_list:has(.js_amount)"
+            >
+                <li data-only="click" data-increase="1">
+                    <a tabindex="-1"><i class="fa fa-plus" /> Add one event</a>
+                </li>
+                <li data-only="click" data-increase="-1">
+                    <a tabindex="-1"><i class="fa fa-minus" /> Remove one event</a>
+                </li>
+            </div>
+        </xpath>
+    </template>
+</odoo>
```

Comment:
You need to fix this template and adapt it to the new format in v13 to make it work. It's broken as you said.

Reply:
any hint where the problem lies with this template?

Reply:
Well.. you'll have to take a look at how website snippets are added in upstream `website` for v13 and try to imitate that method here.